core/ppu.cpp

```diff
@@ -74,7 +74,7 @@ bool PPU::isBgWinEnabled() { return checkBit(get_lcdc(), 0); }
 
 void PPU::step(u8 cpuCyclesElapsed) {
 
-  if (!isLCDEnabled()) { return; }
+  // if (!isLCDEnabled()) { return; }
```

Comment:
Commented out while debugging. Might not be necessary now that we're reading 16-bit values correctly.